workspace/viewer/PdfJsViewer.ts

```diff
@@ -478,29 +480,32 @@ export class PdfJsViewer {
       canvas.removeClass("shadow");
     }
     const canvasOffset = canvas.position();
-    const textLayer = $(".textLayer", this.jqRoot);
+    const textLayer = parent.find(".textLayer");
     textLayer.css({
       top : canvasOffset.top,
       left : canvasOffset.left
     });
   }
 
   resetCanvas() {
-    if (this.pdfPageView) {
-      this.pdfPageView.destroy();
+    if (this.loadedPages.length != 0) {
```

Comment:
What's the purpose of this `if`? If array is empty we will just make no loops below without additional ifs.

Reply:
Done.